SignalMessaging/utils/OWSPreferences.h

```diff
@@ -58,8 +58,8 @@ extern NSString *const OWSPreferencesKeyEnableDebugLog;
 - (BOOL)hasDeclinedNoContactsView;
 - (void)setHasDeclinedNoContactsView:(BOOL)value;
 
-- (void)setIOSUpgradeNagVersion:(NSString *)value;
-- (nullable NSString *)iOSUpgradeNagVersion;
```

Comment:
I think we want to retain the existing nag behavior for people on iOS9+, right?  We don't want people running super-old versions of Signal regardless of their platform.

Though we may consider disabling the "old nag" for the few people running iOS8, since they may not be able to update to the "latest" version.

Reply:
I think you're thinking of the _signal update_ nag implemented in `AppUpdateNag`.  That remains in place.  I'm just removing the state we used to use to throttle the _iOS update_ nag.

Reply:
You're correct - sorry for the confusion.

LGTM!